config.yml.erb

```diff
@@ -214,6 +214,10 @@ firebase_max_table_rows: 20000
 firebase_max_record_size: 4096
 firebase_max_property_size: 4096
 
+# Azure keys for speech service
```

Comment:
config.yml.erb acts as the defaults across all environments, so if you want them in every environment, you should only need to add them here, and not in the development/staging/test/production/levelbuilder yml files. (It would still work the way it is right now, it's just redundant.)